service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -80,11 +81,45 @@
     @Override
     public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request) {
```

Comment:
I've created a new user-story to track these comments : 

https://rally1.rallydev.com/#/59288076735d/detail/userstory/67067764948